src/tools/mesh_tester.cpp

```diff
@@ -79,10 +79,13 @@ using UniformGrid3 = spin::UniformGrid<int, 3>;
 using Vector3 = primal::Vector<double, 3>;
 using Segment3 = primal::Segment<double, 3>;
 
-enum RuntimePolicy { seq = 0,
-                     raja_seq = 1,
-                     raja_omp = 2,
-                     raja_cuda = 3};
+enum RuntimePolicy
+{
+  seq = 0,
```

Comment:
How do we feel about alignment of operators and assignments.
I left this off since it would introduce even more changes to the code.

In general , we have not been consistent about this in our current codebase (perhaps it is flagged as `ignore` in our current uncrustify)-- some files are aligned, others are not.

Reply:
I think I tend to align especially for things in the same block of code, but, I don't have a strong feeling on this and it may be OK to be more _loose_ on that.